IHP/Job/Dashboard.hs

```diff
@@ -0,0 +1,530 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE GADTs #-}
+
+{-|
+Module: IHP.Job.Dashboard
+Description:  Auto-generate a dashboard for job types
+Maintainer: zac.wood@hey.com
```

Comment:
Just an idea: [we could a code owner file](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners), then github could automatically request e.g. code reviews from you when someone touches the `Dashboard.hs` file :)